Base/Python/slicer/util.py

```diff
@@ -1091,3 +1091,17 @@ def onClick(caller,event):
   up()
   interactor.SetShiftKey(0)
   interactor.SetControlKey(0)
+
+def getFilesInDirectory(directory, absolutePath=True):
+  """ Collect all files in a directory and its subdirectories in a list
+  """
+  import os
+  allFiles=[]
+  for root, subdirs, files in os.walk(directory):
+    for file in files:
```

Comment:
Since `file`is a python built-in function, consider using `fileName`. See https://docs.python.org/2/library/functions.html#file